core/src/main/java/com/novoda/merlin/MerlinBuilder.java

```diff
@@ -22,13 +19,13 @@
 
 public class MerlinBuilder {
 
-    private BindListener merlinOnBinder;
-    private ConnectListener merlinConnector;
-    private DisconnectListener merlinDisconnector;
+    private BindCallbackManager merlinOnBinder;
```

Comment:
`CallbackManager` is more explicit than `Listener` and highlights that it is responsible for sending one event from the `Receiver` etc to all of the internal `Registerables`.